src/test/resources/fixtures/hmh/hmh-mi/development/caliperEventAssessmentStarted.json

```diff
@@ -0,0 +1,35 @@
+{
+  "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "type":"AssessmentEvent",
+  "id":"urn:uuid:27734504-068d-4596-861c-2315be33a2a2",
+  "actor":{
+    "id":"urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
+    "type":"Person"
+  },
+  "action":"Started",
+  "object":{
+    "id":"urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
+    "type":"Assessment",
+    "name":"MathInventory",
+    "maxAttempts":1,
+    "maxSubmits":1,
+    "maxScore":25.0,
+    "version":"1.0"
```

Comment:
fyi - `version` is an optional field - "A string value that designates the current form or version of the Assessment." 

Just letting you know, if you don't need `version` and are just including it because you've seen other caliper json files having it, there's no need to include it - only include it if you want it captured.
I am aware it's in other 'caliper assessment started event' files that have been submitted but I think that might be folks following the pattern that has been slowly established.